framework/commercetools/lib/normalize.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  CommercetoolsProducts,
+  CommerceToolsProductPrice,
+  Product,
+  ProductVariant,
+  CommercetoolsProductVariant,
+  ProductPrice,
+  Images,
+  ProductImage,
+} from '../types/product'
+import type {
+  Cart,
+  CommercetoolsCart,
+  CommercetoolsLineItems,
+  LineItem,
+} from '../types/cart'
+
+function setProductPrice(prices: CommerceToolsProductPrice): ProductPrice {
```

Comment:
This is a `price` and not `prices`, isn't it?

Reply:
This is not setting anything, is just getting the formatted Price. I would rename the method.